sunpy/sun/models.py

```diff
@@ -20,6 +20,7 @@
 
 import pandas
 from astropy.units import Quantity
+from astropy.table import QTable
 import sunpy.sun.constants as con
 
```

Comment:
can you add an `__all__` list to this file with just the models in it please?

Reply:
Made the changes, could you tell me if that is what you meant ?